Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift

```diff
@@ -46,8 +49,34 @@ extension ConnectionConfiguration {
                 }
             })
         }
-
+        
         return authCallbackWrapper
     }
-
+    
+    private func createObjCAuthCallbackWrapper(_ authCallback: @escaping ObjCAuthCallback) -> (ARTTokenParams, @escaping (ARTTokenDetailsCompatible?, NSError?) -> ()?) -> () {
```

Comment:
Should it be `createAuthCallback(from: objCAuthCallback)`? Just a suggestions :)

Reply:
I don't like this naming too, but we have to be sure what this method is for, and this one is for Objective-C

Reply:
But the method parameter being `@escaping ObjCAuthCallback` ensures this already. 

My **thought** when I first read the method name was not what the method does: It is creating wrapper for AuthCallback so it can be used in Objective-C (wrong). Instead of creating an AuthCallback wrapper for an Objective-C AuthCallback (the actual method implementation). But, then I thought, the signature already explains this behaviour already.

just a minor point

Reply:
of course, I could name it the same since they have different argument types - I forgot it's swift 🙉
I'll rename them to `createAuthCallback`

Thanks

Reply:
commit f6249cfefb0be006573966890e1154e564333068

Reply:
Cool